### ISSUE TITLE
Auth user can change status of Session submission

### ISSUE THREAD

**Issue by jdearie** [Drupal4Gov/Drupal-GovCon-2017#329]
A user should not be able to change the status of their session submission.  When I go to edit a session (as an auth user) I've submitted, I can change the status

![image](https://cloud.githubusercontent.com/assets/5174534/25289029/ff420924-2695-11e7-8c16-86b588ac814d.png)

**Comment by jdearie** [comment]
@aweingarten - super important this one.

**Comment by aweingarten** [comment]
Fixed!

**Comment by timwood** [comment]
Issue still exists at /node/ID/edit as the hook implemented in https://github.com/drupal-cod/program/commit/a851b6021fe9528750b139c2a2f4d1fe90d596f2 only affects /node/add/session. 'session_form_node_session_form_alter' vs session_form_node_session_**edit**_form_alter'

Thanks @sarahjean

**Comment by mikemadison13** [comment]
accidentally duped at https://github.com/Drupal4Gov/Drupal-GovCon-2017/issues/376

**Comment by mikemadison13** [comment]
i've addressed this here: https://github.com/drupal-cod/program/pull/8 by swapping from the hook hook_form_FORM_ID_alter to just a plain hook_form_alter with an if statement in it per @sarahjean's recommendation (good call). 

the issue was that the form alter in the code was only firing on the creation (and not the editing) of the session nodes.

i'll work on getting this merged (and then we should have it on our next deployment). 

cc @aweingarten

**Comment by mikemadison13** [comment]
update, @japerry merged the PR for me so this should be able to be verified locally now @timwood

**Comment by timwood** [comment]
@mikemadison13 I tried for a while tonight to test this locally so we can close out this ticket, but was unsuccessful in getting my local environment to update the cod program module. I tried:

`blt local:sync ; blt local:refresh` (one of which runs the blt setup:composer:install task)

manually running `composer install` 

checking `composer status -v` and `composer outdated` the first command reported there was a version variation in our composer.json, which I corrected locally with `composer require drupal-cod/program:master` and just to be sure `composer update drupal-cod/program`.
```
You have version variations in the following dependencies:
docroot/modules/contrib/program:
    From master to dev-master
```

The only things I didn't try was `composer nuke` and manually going into the docroot/modules/contrib/program folder to run `git pull`

Thoughts?
